### PR TITLE
layout: Rename the Download to Download index.

### LINKED ISSUE #2618: templates: download index
When offering download for index files, the button should say "Download index" rather than "Download" in order to avoid misunderstandings.  This would give "List files - Download index".

Example: http://opendata-dev.cern.ch/record/12100

### PR DESCRIPTION
**Pull request by psaiz**
Closes #2618

### CODE REVIEW

File: cernopendata/modules/theme/assets/semantic-ui/js/records/components/FileTable.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ export default function FileTable({ items }) {
                   </Button>
                 )}
                 <Button as="a" icon size="mini" primary {...downloadProp}>
-                  <Icon name="download" /> Download
```
Comment (tiborsimko) [inline cernopendata/modules/theme/assets/semantic-ui/js/records/components/FileTable.js:83]: The button label should be dynamic:

- for data files attached via file indexes, such as https://opendata.cern.ch/record/1, the button should say "Download index";

- for data files attached directly, such as https://opendata.cern.ch/record/5500, the button should say "Download".

(The index files can be recognised by having `files.type` field value "index.json" or "index.txt" in the record JSON.)

Comment (katilp) [reply cernopendata/modules/theme/assets/semantic-ui/js/records/components/FileTable.js:83]: @psaiz The dynamic button label would be indeed needed: we have had to explain this separately to the OD users as they got confused. Now we will have more files that are small enough to be downloaded so this is even more important.

File: cernopendata/modules/theme/assets/semantic-ui/js/records/components/FileTable.js [hunk 2]
```diff
@@ -60,6 +60,7 @@ export default function FileTable({ items }) {
                   },
                 }
               : { href: getFileUri(file.key) };
+          const label= file.type=='index.txt' ? 'Download index' : 'Download';
```
Comment (zzacharo) [inline cernopendata/modules/theme/assets/semantic-ui/js/records/components/FileTable.js:63]: could that be also `index.json`?

Comment (psaiz) [reply cernopendata/modules/theme/assets/semantic-ui/js/records/components/FileTable.js:63]: THe index.json entries seem to be filtered out at some point (I haven't figure it out where exactly).  See for instance [this entry](https://opendata.cern/record/28003), which contains a normal file and a  file set:
Screenshot 2024-03-27 at 16 27 14

If we look at the entry on opensearch, it has three files:
Screenshot 2024-03-27 at 16 28 00

**Comment by psaiz** [comment]
This has been addressed in the cernopendata_portal repo

**Pull request closed without merging**